Application/Telegram/Models/UserInfo.cs

```diff
@@ -8,6 +9,13 @@
 
 namespace Application.Telegram.Models
 {
+    public class DateInfo
+    {
+        public int StartDay { get; set; } 
+        public int StartMonth { get; set; } 
+        public int EndMonth { get; set; } 
+        public int EndDay { get; set; } 
+    }
```

Comment:
I think we do not need CreateBookingRequest object, because at the same time user cannot book and create vacation so we can create one class for creating booking and vacation objects and the reason for creating this class is for collecting user inputs. We can add mapId property to this class.

Reply:
I added seperate property for mapId